src/Okta.Sdk.IntegrationTests/DomainScenarios.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.Threading.Tasks;
+using FluentAssertions;
+using Xunit;
+
+namespace Okta.Sdk.IntegrationTests
+{
+    public class DomainScenarios
+    {
+        [Fact]
+        public async Task CreateDomain()
+        {
+            var oktaClient = TestClient.Create();
+            var domainsClient = oktaClient.Domains;
+            var domainName = $"domain{new Random().Next(1000000)}.example.com";
+            var newDomain = new Domain
+            {
+                DomainName = domainName,
+                CertificateSourceType = DomainCertificateSourceType.Manual,
+            };
+
+            var createDomainResult = await domainsClient.CreateDomainAsync(newDomain);
```

Comment:
Is the use of `var` the established standard?  It would be nice to know what this type actually is by looking at the code without having to look at the definition of CreateDomainAsync.

Reply:
This makes whole code cleaner, I think using `var` is acceptable for small internal functions like these ITs.